server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -0,0 +1,106 @@
+
+=== KeycloakBackup Custom Resource
+
+{project_operator} provides automatic backups with manual restore in 3 modes:
+
+* One time backups to a local Persistent Volume.
+* One time backups to Amazon S3 storage.
+* Periodic backups to Amazon S3 storage.
+
+The Operator uses `KeycloakBackup` Custom Resource to trigger a backup Job (or a `CronJob` in case of Periodic Backups) and reports back its status. The CR has the following structure:
+
+.`KeycloakBackup` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakBackup
+metadata:
+  name: <CR Name>
+spec:
+  aws:
+    # Optional - used only for Periodic Backups
```

Comment:
Maybe `# Optional - used only for Periodic Backups. Follows usual crond syntax (e.g. use "0 1 * * *") to perform the backup every day at 1 AM` ? If the expected syntax is different, can you provide an example value?

Reply:
Fxied.